packages/manager-react-components/src/components/tags-tile/tags-tile.component.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { OdsLink } from '@ovhcloud/ods-components/react';
+import { ODS_ICON_NAME } from '@ovhcloud/ods-components';
+import { TagsList } from '../tags-list';
+import './translations';
+import { ManagerTile } from '../content';
+
+export interface TagsTileProps {
+  tags: { [key: string]: string };
+  displayInternalTags?: boolean;
+  onEditTags: () => void;
+}
+
+export const TagsTile = React.forwardRef<HTMLOdsModalElement, TagsTileProps>(
```

Comment:
Why `React.forwardRef<HTMLOdsModalElement,` ?
Actually you don't have a modal and you don't even pass the ref so the correct type might be `React.FC<TagsTileProps>`